resop.mod

```diff
@@ -0,0 +1,27 @@
+set V; # set of graph nodes
+set S; # set of services
+set E; # graph edges
+#set A; #set of VFs
+set ES; # service graph edges
+param n;
+set N :={1..n};
```

Comment:
Since the graph nodes is the union of the PoAs, servers, switches, and robots; lets define it as such:

```ampl
set N = R union r union w union s;
```

And check if AMPL is case-sensitive, otherwise, servers' set `s`, and services' set `S` will be confused.